script/data.js

```diff
@@ -0,0 +1,65 @@
+class Book {
```

Comment:
we are missing the render function, I can see your `getBook` function, but one of the project requirement is to have a render function that loops through the book array and displays each book on the page.
Kindly create the render function as we both know all other students follow this project guideline too. 👍 
![image](https://user-images.githubusercontent.com/31897434/85427772-2207ed00-b574-11ea-8f7b-dd9f2164572b.png)
https://www.theodinproject.com/courses/javascript/lessons/library